.github/workflows/generate_ipynb_files.yml

```diff
@@ -13,6 +13,7 @@ jobs:
       GH_SERVICE_ACCOUNT_NAME: "tflite-bot"
       GH_SERVICE_ACCOUNT_EMAIL: "tflite-bot@antmicro.com"
       GH_SERVICE_ACCOUNT_TOKEN: ${{ secrets.GH_SERVICE_ACCOUNT_TOKEN }}
+      CLONE_REPO_SED_CMD: $(sed -i '1s/^/!git clone https:\/\/github.com\/antmicro\/tensorflow-arduino-examples.git .\/ \n/' build.sh)
```

Comment:
can we have something like `echo -e "git clone .... \n$(cat build.sh)" > build.sh` instead? It's so much simpler

Reply:
and maybe add `sed -i 's/^/!/' build.sh` to the same variable. Also rename to `PREPARE_BUILD_SH`

Reply:
will the dot work? Is it a guaranteed empty directory?